pymc/tests/test_model.py

```diff
@@ -356,6 +356,9 @@ def test_missing_data(self):
 
         assert m["x2_missing"].type == gf._extra_vars_shared["x2_missing"].type
 
+        # The dtype of the merged observed/missing deterministic should match the RV dtype
+        assert m.deterministics[0].type.dtype == str(X.dtype)
```

Comment:
Sorry @michaelosthege, I've been busy with school stuff. My guess is that is line is what causes the test to fail?